swift/test/Ice/scope/Test.ice

```diff
@@ -156,26 +156,54 @@ module Test
 ["swift:module:Test:Inner"]
 module Inner
 {
+    module Test::Inner2
+    {
+        interface I
+        {
+            Test::S opS(Test::S s1, out Test::S s2);
+            Test::SSeq opSSeq(Test::SSeq s1, out Test::SSeq s2);
+            Test::SMap opSMap(Test::SMap s1, out Test::SMap s2);
 
-module Test
-{
+            Test::C opC(Test::C c1, out Test::C c2);
+            Test::CSeq opCSeq(Test::CSeq c1, out Test::CSeq c2);
+            Test::CMap opCMap(Test::CMap c1, out Test::CMap c2);
+
+            void shutdown();
+        }
+    }
+}
 
-module Inner2
+// Check that it's okay for 'swift:module' to have different 'prefix' arguments.
```

Comment:
I am sure we can test this "feature" with a simpler test.

Reply:
Well, I also want to make sure that it gets mapped to the correct place as well.
Not just that "the compiler doesn't issue an error anymore".

Because we don't have a test for the swift:module anywhere, and scope felt like an adequate place to put it.
Since the point of the metadata is changing the mapped scopes.